Arrays.js

```diff
@@ -0,0 +1,137 @@
+// }
+
+function doubleArray(arr) {
+  
+   return arr.concat(arr)
+ }
+   console.log(doubleArray(['Ace', 10, true]))
+   console.log(doubleArray([0, 1, 2, 3, 4, 5]))
+   console.log(doubleArray([]))
+
+
+   // 3 EX
+   // Returns an array of positive numbers from the specified array in original order
+// @param {array} arr
+// @return {array}
+// @example
+// [ 0, 1, 2, 3, 4, 5 ] => [ 1, 2, 3, 4, 5 ]
+// [-1, 2, -5, -4, 0] => [ 2 ]
+// [] => []
+// */
+// function getArrayOfPositives(arr) {
+// return
+// }
+// /**
+function getArrayOfPositives(arr) {
+   return arr.filter(item => item > 0)
+   }
+   console.log(getArrayOfPositives([0, 1, 2, 3, 4, 5]))
+   console.log(getArrayOfPositives([-1, 2, -5, -4, 0]))
+   console.log(getArrayOfPositives([]))
+
+// 4 EX
+//  Returns the array with strings only in the specified array (in original order)
+// @param {array} arr
+// @return {array}
+// @example
+// [ 0, 1, 'cat', 3, true, 'dog' ] => [ 'cat', 'dog' ]
+// [ 1, 2, 3, 4, 5 ] => []
+// [ 'cat, 'dog', 'raccon' ] => [ 'cat', 'dog', 'racoon' ]
+// */
+// function getArrayOfStrings(arr) {
+// return
+// }
+// /**
+
+function getArrayOfStrings(arr) {
+  
+      return arr.filter(el => typeof el === 'string')
+           }
+     
+   console.log(getArrayOfStrings([0, 1, 'cat', 3, true, 'dog']))
+   console.log(getArrayOfStrings([1, 2, 3, 4, 5]))
+   console.log(getArrayOfStrings([ 'cat', 'dog', 'raccon' ]))
+
+//  Returns the array with numbers only in the specified array (in original order)
+   function getArrayOfStrings1(arr) {
+  
+      return arr.filter(el => typeof el === 'number')
+           }
+     
+   console.log(getArrayOfStrings1([0, 1, 'cat', 3, true, 'dog']))
+   console.log(getArrayOfStrings1([1, 2, 3, 4, 5]))
+   console.log(getArrayOfStrings1([ 'cat', 'dog', 'raccon' ]))
+// 5 EX Returns the array of useprcase strings from the specified array
+// @param {array} arr
+// @return {array}
+// @example
+// [ 'permanent-internship', 'glutinous-shriek', 'multiplicative-elevation' ] => [ 'PERMANENT-INTERNSHIP', 'GLUTINOUS-SHRIEK', 'MULTIPLICATIVE-ELEVATION' ]
+// [ 'a', 'b', 'c', 'd', 'e', 'f', 'g' ] => [ 'A', 'B', 'C', 'D', 'E', 'F', 'G' ]
+// */
+// function getUpperCaseStrings(arr) {
+// return
+// }
+// /**
+
+function getUpperCaseStrings(arr) {
+   const newArray = arr.map(el => {return el.toUpperCase()})
+   console.log(newArray)
+   }
+   getUpperCaseStrings(['permanent-internship', 'glutinous-shriek', 'multiplicative-elevation'])
+   getUpperCaseStrings([ 'a', 'b', 'c', 'd', 'e', 'f', 'g'])
+
+// 6 EX
+// Returns the array of string lengths from the specified string array.
+// @param {array} arr
+// @return {array}
+// @example
+// [ '', 'a', 'bc', 'def', 'ghij' ] => [ 0, 1, 2, 3, 4 ]
+// [ 'angular', 'react', 'ember' ] => [ 7, 5, 5 ]
+// */
+// function getStringsLength(arr) {
+// return
+// }
+
+ function getStringsLength(arr) {
+   const newArray = arr.map((word) => {return word.length})
+   console.log(newArray)
```

Comment:
Good job, but you also could write like this 
`function getStringsLength(arr) {
  return arr.map(word => word.length);
}`

Reply:
Ok, thanks